examples/bri-3/src/bri/identity/bpiAccounts/models/bpiAccount.ts

```diff
@@ -1,29 +1,23 @@
+import { AutoMap } from '@automapper/classes';
 import { BpiSubjectAccount } from '../../bpiSubjectAccounts/models/bpiSubjectAccount';
 
 export class BpiAccount {
-  private _id: string; // TODO: Add uuid after #491
-  private _nonce: number;
-  private _ownerBpiSubjectAccounts: BpiSubjectAccount[];
+  @AutoMap()
+  id: string; // TODO: Add uuid after #491
 
-  constructor(id: string, ownerBpiSubjectAccounts: BpiSubjectAccount[]) {
-    this._id = id;
-    this._nonce = 0;
-    this._ownerBpiSubjectAccounts = ownerBpiSubjectAccounts;
-  }
-
-  public get id(): string {
-    return this._id;
-  }
+  @AutoMap()
+  nonce: number;
 
-  public get nonce(): number {
-    return this._nonce;
-  }
+  @AutoMap(() => [BpiSubjectAccount])
+  ownerBpiSubjectAccounts: BpiSubjectAccount[];
 
-  public get ownerBpiSubjectAccounts(): BpiSubjectAccount[] {
-    return this._ownerBpiSubjectAccounts;
+  constructor(id: string, ownerBpiSubjectAccounts: BpiSubjectAccount[]) {
+    this.id = id;
+    this.nonce = 0;
```

Comment:
why are you adding a `nonce` here? @Manik-Jain ... `nonce` is associated with a BpiAccount